docs/reference/api/api.mdx

```diff
@@ -20,10 +20,26 @@ The unique name of this API within the app. Subsequent calls to `api` with the s
 
 Additional options when creating the API.
 
-| Properties                                                                                                                     |
-| ------------------------------------------------------------------------------------------------------------------------------ |
-| **path** optional `string` <br/> base path for all routes in the API.                                                          |
-| **middleware** optional `HttpMiddleware` or `HttpMiddleware[]` <br/> Middleware to apply to all routes and methods of the API. |
+| Properties                                                                                                                                        |
+| ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **path** optional `string` <br/> base path for all routes in the API.                                                                             |
+| **middleware** optional `HttpMiddleware` <br/> Middleware to apply to all routes and methods of the API.                                          |
```

Comment:
Should this still be HttpMiddleware or HttpMiddleware[]?

Reply:
Intentionally removed the array as support for this has been removed (the last element in the args in the is now options rather than middleware being variadic.

Reply:
fixed.